include/dpp/discordvoiceclient.h

```diff
@@ -50,14 +50,18 @@
 #include <functional>
 #include <chrono>
 
-
-
 struct OpusDecoder;
 struct OpusEncoder;
 struct OpusRepacketizer;
 
 namespace dpp {
 
+inline constexpr uint16_t AUDIO_TRACK_MARKER = 0xFFFF;
+
+inline constexpr int AUDIO_OVERLAP_SLEEP_SAMPLES = 30;
+
+inline constexpr size_t SEND_AUDIO_RAW_MAX_LENGTH = 11520;
```

Comment:
Hold on, how the naming should be? `dpp::SEND_AUDIO_RAW_MAX_LENGTH` doesn't seems right. I guess I will change it to snake case for this variable only